modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/management_bar/components/filters/AutocompleteFilter.js

```diff
@@ -71,24 +83,34 @@ function getOdataString(value, key, selectionType) {
 		? `${key}/any(x:${value
 				.map(
 					(v) =>
-						`(x eq ${
+						`(x ${exclude ? 'ne' : 'eq'} ${
 							typeof v.value === 'string'
 								? `'${v.value}'`
 								: v.value
 						})`
 				)
-				.join(' or ')})`
-		: `${key} eq ${
+				.join(exclude ? ' and ' : ' or ')})`
+		: `${key} ${exclude ? 'ne' : 'eq'} ${
 				typeof value[0].value === 'string'
 					? `'${value[0].value}'`
 					: value[0].value
 		  }`;
 }
-
-function AutocompleteFilter(props) {
+function AutocompleteFilter({
+	actions,
+	apiURL,
+	id,
+	inputPlaceholder,
+	itemKey,
+	itemLabel: itemLabelProp,
+	selectionType,
+	value: valueProp,
+}) {
 	const [query, setQuery] = useState('');
 	const [search, setSearch] = useState('');
-	const [selectedItems, setSelectedItems] = useState(props.value || []);
+	const [selectedItems, setSelectedItems] = useState(
+		(valueProp && valueProp.items) || []
```

Comment:
Maybe [nullish coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) together with [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) could make this a little more compact:

```js
const [selectedItems, setSelectedItems] = useState(valueProp?.items ?? []);
```